rpm/__init__.py

```diff
@@ -38,15 +38,19 @@ def get_system_sitepackages_and_suffixes() -> List[Dict[str, List[str]]]:
     def get_sitepackages_and_suffixes(interpreter):
         script = textwrap.dedent(
             """
-            import importlib
-            import importlib.machinery
             import json
             import site
+            try:
```

Comment:
By convention there should be an empty line between imports and code (I realize the code in this case is a "conditional" import, but it's not considered as one).